sources/Interop/Windows/Windows/shared/windef/RECT.cs

```diff
@@ -23,4 +25,19 @@ public partial struct RECT
     /// <include file='RECT.xml' path='doc/member[@name="RECT.bottom"]/*' />
     [NativeTypeName("LONG")]
     public int bottom;
+
+    // type 'Rectangle' is from System.Drawing.Primitives which is from the base shared framework.
+    // as such these operators *should* be safe to include here.
+    public static implicit operator Rectangle(RECT rectangle)
+        => Rectangle.FromLTRB(rectangle.left, rectangle.top, rectangle.right, rectangle.bottom);
+
+    public static explicit operator RECT(Rectangle rectangle)
+        => new
+        {
```

Comment:
nit: `new()` is one of my least favorite language features and I'm pretty sure it's blocked in the editorconfig, so I expect this will cause CI to fail.